app/sdil/controllers/CopackedController.scala

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2017 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package sdil.controllers
+
+import javax.inject.Inject
+
+import play.api.data.Forms.{boolean, optional, single}
+import play.api.data.{Form, Mapping}
+import play.api.i18n.{I18nSupport, MessagesApi}
+import play.api.mvc.{Action, AnyContent}
+import sdil.config.FormDataCache
+import uk.gov.hmrc.http.cache.client.SessionCache
+import uk.gov.hmrc.play.frontend.controller.FrontendController
+import views.html.softdrinksindustrylevy.register
+
+import scala.concurrent.Future
+
+class CopackedController @Inject()(val messagesApi: MessagesApi) extends FrontendController with I18nSupport {
+
+  val cache: SessionCache = FormDataCache
+
+  private val copackedForm = Form(single(
+    "isCopacked" -> booleanMapping))
+
+  def display = Action { implicit request =>
+    Ok(register.copacked(copackedForm))
+  }
+
+  def submit = Action.async { implicit request =>
+      copackedForm.bindFromRequest.fold(
+        formWithErrors => {
+          Future.successful(
+            BadRequest(register.copacked(formWithErrors)))
+        },
+        validFormData => {
+          if (validFormData){
+            Redirect(routes.LitreageController.show("copackedVolume"))
+          }
+          else{
+            Redirect(routes.ImportController.display())
+          }
+        })
+    }
+  private lazy val booleanMapping: Mapping[Boolean] =
```

Comment:
This is a dupe of the one in the SDILController. Both should probably be moved either to the controller package or the forms package.